### ISSUE TITLE
How to reproduce the results in the paper (instant-ngp)?

### ISSUE THREAD

**Issue by Pamiiii** [NVIDIAGameWorks/kaolin-wisp#42]
I train the ngp-nerf with conmand
`python3 app/main.py --config configs/ngp_nerf.yaml --multiview-dataset-format standard --mip 0 --dataset-path data/lego`
I only changed epochs to 100
The PSNR of result is only 24.52 in the lego dataset.
How can i reproduce the results in the paper (instant-ngp)
<img width="795" alt="image" src="https://user-images.githubusercontent.com/89392940/189891254-822619e6-143a-4472-be16-308db867aa3f.png">

**Comment by c-he** [comment]
Same question here! I trained on the lego scene with the provided configuration file `configs/ngp_nerf_interactive.yaml`, but the results look a bit blurry and not as good as what I got from instant-ngp.
![Screenshot from 2022-11-08 21-07-19](https://user-images.githubusercontent.com/15975829/200719715-df15662d-f6ab-4993-9f1c-e92885b7505d.png)

**Comment by Hippogriff** [comment]
I still cannot reproduce results using the latent version of the wisp. Does anyone know the correct hyper parameters?

**Comment by Hippogriff** [comment]
@orperel has there been any update on how to reproduce the results from the original instant-NGP paper? The current version of wisp doesn't match the performance.

**Comment by orperel** [comment]
Hi! We worked around this issue, but we need to further clean this MR before we can merge it.
Are you blocked for ICCV? If so, we can post the dirty MR until this effort is done.

Otherwise, the metrics in the [NeRF table](https://github.com/NVIDIAGameWorks/kaolin-wisp/tree/main/app/nerf#evaluation-metrics) should be up to date.
(after we merge PSNR on lego should go above > 35.0)

**Comment by Hippogriff** [comment]
Yes, please post/share the required changes needed. I do indeed need it for ICCV.

**Comment by Hippogriff** [comment]
@tovacinni Has there been any update on this? I would really appreciate if the results from the original papers can be reproduced. Otherwise, it is hard to use this framework to do experiments related to publications. Thanx for your hard work.

**Comment by Hippogriff** [comment]
@tovacinni If you have specific values for these hyper parameters, that will be great. Thanx.
